analytics/index.ts

```diff
@@ -4,6 +4,13 @@ const formatEventName = (eventName: string) =>
   eventName.split(' ').join('-').toLocaleLowerCase();
 
 export const FA_EVENTS = {
+  CHAT: {
+    NEW: 'click/chat/new',
+    NAVBAR: 'click/chat/navbar',
+    WIZARD: 'click/chat/wizard',
+    OPEN: (id: string) => `click/chat/${id}`,
+    OPEN_WIZARD: (id: string) => `click/chat/wizard/${id}`,
```

Comment:
should we track when there is multiple messages or we check that on the db?

Reply:
I don't know what the goal is on tracking if there were multiple messages in the chat. 

Maybe we could check if there is one or more messages from the user to know if the users are chatting with the bot?